tests/unit/TableTest.php

```diff
@@ -274,27 +274,18 @@ public function testDeleteColumn()
         $this->assertNull($r);
     }
 
-    protected function createTestTable(string $tableName,bool $hasAge = true):Morebec\YDB\Table
+    protected function createTestTable(string $tableName,array $additionalColumns = []):Morebec\YDB\Table
     {
-        $schema = $this->createTestTableSchema($hasAge);
-
-        return $this->database->createTable(
-            new TableSchema($tableName,$schema)
-        );
-    }
-
-    protected function createTestTableSchema(bool $hasAge)
-    {
-        $schema = [
+        $baseSchema = [
```

Comment:
Could you rename this to baseColumns? These are really just columns then a full schema definitions